openlibrary/plugins/openlibrary/js/jquery.fadeToggle.js

```diff
@@ -0,0 +1,5 @@
+// ADD FADE TOGGLE
+// COURTESY: Karl Swedberg http://www.learningjquery.com/2006/09/slicker-show-and-hide
+export default function(speed, easing, callback) {
```

Comment:
Since this is no longer evident from context:

```suggestion
/**
 * @this {jQuery}
 */
export default function(speed, easing, callback) {
```